eras/alonzo/impl/src/Cardano/Ledger/Alonzo/TxInfo.hs

```diff
@@ -218,13 +241,13 @@ txInfoOut ::
     HasField "datahash" (Core.TxOut era) (StrictMaybe (DataHash c))
   ) =>
   Core.TxOut era ->
-  Maybe PV1.TxOut
+  Either TranslationError PV1.TxOut
 txInfoOut txout =
   let val = getField @"value" txout
       datahash = getField @"datahash" txout
-   in case transTxOutAddr txout of
-        Just ad -> Just (PV1.TxOut ad (transValue @(Crypto era) val) (transDataHash datahash))
-        Nothing -> Nothing
+   in case transTxOutAddr txout ByronOutputInContext of
+        Right ad -> Right (PV1.TxOut ad (transValue @(Crypto era) val) (transDataHash datahash))
+        Left err -> Left err
```

Comment:
```suggestion
   in case transTxOutAddr txout of
        Just ad -> Right (PV1.TxOut ad (transValue @(Crypto era) val) (transDataHash datahash))
        Nothing -> Left ByronOutputInContext
```